content/courses/onchain-development/intro-to-anchor-frontend.md

```diff
@@ -271,16 +394,17 @@ and listen to events. You can also
 To create the `Program` object, first import `Program` and `Idl` from
 `@coral-xyz/anchor`. `Idl` is a type you can use when working with Typescript.
 
-Next, specify the `programId` of the program. We have to explicitly state the
-`programId` since there can be multiple programs with the same IDL structure
-(i.e. if the same program is deployed multiple times using different addresses).
 When creating the `Program` object, the default `Provider` is used if one is not
 explicitly specified.
 
+To enable type support, import the types to your project from `/target/types`
+present in your anchor project, and declare the type for the program object.
+
 All together, the final setup looks something like this:
 
 ```typescript
 import idl from "./idl.json";
+import type { CounterContract } from "@/types";
```

Comment:
Call it a program not a contract. This is mentioned in CONTRIBUTING, also https://solana.com/docs/terminology#onchain-program